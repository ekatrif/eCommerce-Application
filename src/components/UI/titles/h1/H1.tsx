import React from 'react';
import classes from './H1.module.css';

interface H1Props {
  text?: string;
}

const H1: React.FC<H1Props> = ({ text }) => {
  return <h1 className={classes.h1}>{text}</h1>;
};

export default H1;
