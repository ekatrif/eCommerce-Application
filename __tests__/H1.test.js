import React from 'react';
import renderer from 'react-test-renderer';
import H1 from '../src/components/UI/titles/h1/H1';

test('H1 title renders correctly', () => {
  const tree = renderer
    .create(<H1 text="Login" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
