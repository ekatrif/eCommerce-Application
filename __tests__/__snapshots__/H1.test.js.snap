// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 title renders correctly 1`] = `
<h1
  className="h1"
>
  Login
</h1>
`;
